src/core/constants/categories.ts

```diff
@@ -81,6 +81,11 @@ enum StatusCategory {
    */
   PNConnectedCategory = 'PNConnectedCategory',
 
+  /**
+   * Set of active channels and groups has been changed.
+   */
+  PNSubscriptionChangedCategory = 'PNSubscriptionChangedCategory',
```

Comment:
👍🏻 

unfortunately we do not have acceptance test for SubscriptionChanged. As discussed earlier this status emit requirement added few months back

Reply:
@mohitpubnub I added at least unit test to check whether it will be emitted when user subscribed on additional channel.